packages/ui/Slottable/useSlots.js

```diff
@@ -72,11 +72,7 @@ function distribute ({children, ...slots}) {
 			}
 		});
 
-		if (remaining.length > 0) {
-			props.children = remaining;
-		} else {
-			delete props.children;
-		}
+		props.children = remaining.length === 0 ? null : remaining;
```

Comment:
I'm not sure this was strictly necessary but it's shorter than the previous so... win?

Reply:
I've added some inline docs for this. In short, retaining `children` ensure that if the author does `<Wrapped {...props} {...distributed} />` that `props.children` is overwritten by `distributed.children` (since we already distributed the children to other props).

Reply:
That makes sense!